tests/Fixtures/ResetPasswordTestFixtureUserRepository.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the SymfonyCasts ResetPasswordBundle package.
+ * Copyright (c) SymfonyCasts <https://symfonycasts.com/>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace SymfonyCasts\Bundle\ResetPassword\Tests\Fixtures;
+
+use Doctrine\ORM\EntityManagerInterface;
+use SymfonyCasts\Bundle\ResetPassword\Tests\Fixtures\Entity\ResetPasswordTestFixtureUser;
+
+/**
+ * @author Jesse Rushlow <jr@rushlow.dev>
+ * @author Ryan Weaver   <ryan@symfonycasts.com>
+ *
+ * @internal
+ */
+final class ResetPasswordTestFixtureUserRepository
```

Comment:
Does the `ResetPasswordTestFixtureUser` need a custom repository class at all? Or could we remove the `repositoryClass=` part and allow Doctrine to handle it.

Reply:
good catch.. removed user repo fixture